viewing_party/party.py

```diff
@@ -1,23 +1,164 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        return {"title": title, "genre": genre, "rating": rating}
+    return None

```

Comment:
Nice work on this function, very succinct! A suggestion here is for you to change your guard clause to check for **falsy** ` values in `title`, `genre`, and `rating` instead. This because the nature of guard clauses to check for edge cases of our function which is our case is our **falsy** values.
```suggestion
    if not title and not genre and not rating:
        return None
        
    return {"title": title, "genre": genre, "rating": rating}

```